vhosting/resources/vhost.sls

```diff
@@ -19,7 +19,7 @@
 {%- set priority = params.get('priority', 50) %}
 {%- set webroot = params.get('webroot', homedir ~ '/hosts/' ~ name ) %}
 {%- set logdir = params.get('logdir', homedir ~ '/logs' ) %}
-{%- set php = params.get('php', True) %}
+{%- set php = params.get('php', False) %}
```

Comment:
This change seems out of place, why this change?

Reply:
Answered by Alex: Changed the defaults for installing PHP to something more sensible, unrelated to Letsencrypt